Lib/fontbakery/profiles/cff.py

```diff
@@ -154,7 +160,8 @@ def com_adobe_fonts_check_cff_call_depth(cff_analysis):
         for gn in cff_analysis.glyphs_exceed_max:
             yield FAIL, \
                   Message('max-depth',
-                          f'Subroutine call depth exceeded maximum of 10 for glyph "{gn}".')
+                          f'Subroutine call depth exceeded'
+                          f' maximum of 10 for glyph "{gn}".')
```

Comment:
In case you want to streamline the code even further, the `f` prefix is only needed on the line that use the braces `{}` placeholders.

Reply:
yeah, I know. I just feel like it is a bit ugly to not have the `f` prefix in all lines of a multi-line string.